jsonnet/prometheus-operator/alertmanagers-crd.json

```diff
@@ -3261,7 +3264,9 @@
                     "type": "object"
                   },
                   "retention": {
+                    "default": "120h",
                     "description": "Time duration Alertmanager shall retain data for. Default is '120h', and must match the regular expression `[0-9]+(ms|s|m|h)` (milliseconds seconds minutes hours).",
+                    "pattern": "^(0|(([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?(([0-9]+)(u|µ)s)?(([0-9]+)ns)?)$",
```

Comment:
The description does not match the pattern. This validation would allow passing `100µ`, but the description doesn't mention such a possibility.

Reply:
I updated the regex to include from ms to hours. Looks like ns and us is not working